### PR TITLE
AIRNEIS-35 - Dive-In : Protect app secrets

### PR DESCRIPTION
**Pull request by cedricgautier**

### CODE REVIEW

File: docs/dive-in/2024/01/2024-01-17-protect-app-secrets/2024-01-17-protect-app-secrets.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## Context
+
+We are currently building a product in which we use secrets to be able to connect to differennt services. It could be to connect to a database, an email service or a payment service like Stripe.
+
+All these different services provide tokens to connect to them.
+
+## Solution
+
+**The right place to store secrets.**
+To store the secrets in the right place, we need to find a wayy to store our them securely in vault that allows potential identity-based secrets and encryption management system.
```
Comment (KyDev25) [inline docs/dive-in/2024/01/2024-01-17-protect-app-secrets/2024-01-17-protect-app-secrets.md]: Just wayy with one "y"

**Review by KyDev25 — DISMISSED** [review]
Just wayy with one "y". It's easy to understand what the Vault password is. I validate it.

**Review by KyDev25 — DISMISSED** [review]
It's ok!

**Review by BarnabePILLIAUDIN — DISMISSED** [review]
LGTM

File: docs/dive-in/2024/01/2024-01-17-protect-app-secrets/2024-01-17-protect-app-secrets.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- Author: Cedric Gautier
+- Date : 2024-01-17
+
+## Summary
+
+Protect application secrets in our environnements by safely fetching secrets in a secure vault.
+
+## Context
+
+We are currently building a product in which we use secrets to be able to connect to differennt services. It could be to connect to a database, an email service or a payment service like Stripe.
```
Comment (Magali-D) [inline docs/dive-in/2024/01/2024-01-17-protect-app-secrets/2024-01-17-protect-app-secrets.md]: Just a typo on "different"

**Review by Magali-D — DISMISSED** [review]

**Review by Magali-D — APPROVED** [review]

**Review by hallycsse — APPROVED** [review]

**Pull request merged**